2016/proposals/iso-hayate.md

```diff
@@ -0,0 +1,112 @@
+# Title
+
+Online Word2Vec development
+
+## Abstract
+
+Word embedding (as known as Word2Vec) is one of the hot topic in natural language processing research and we can implement it easily using gensim.
+
+However, gensim word2vec implementation has a disadvantage. It cannot update word2vec model after initial training, so we have to retrain word2vec model every time you get the new data.
+
+Therefore, I want to try to add online learning feature to gensim word2vec model.
+
+
+
+## Technical Details
+
+Online learning is a method which uses for stream data and update every time you get data. Vanilla word2vec train the model every time you get the data. The word usage continue to change so we need to update word2vec model constantly.
+
+
+
+## Schedule of Deliverables
+
+### now -  May 24th (not GSoC period)
+
+Understand Mikolov's papers and gensim word2vec architecture.
+
+
+
+### May 25th -  June 7th
+
+
+
+Read [Online Word2Vec for Gensim](http://rutumulkar.com/blog/2015/word2vec/) and gensim's issue #435 and #615 to develop online Word2Vec.
+
+### June 8th - June 21th
+
+
+
+Transcribe online word2vec and implement it at least in local environment.
+
+
+
+### June 22nd - July 5th
+
+
+
+Review gensim word2vec architecture to understand online Word2Vec code in the and resolve AppVeyor error. 
+
+
+
+### July 6th - July 19th
+
+
+

```

Comment:
Please describe HOW will you know if your implementation is correct. What will be your benchmark? What experiment will you run? One full week is needed for an experiment.